src/client/hooks/useThema.ts

```diff
@@ -0,0 +1,22 @@
+import { ChapterTitles } from '../../universal/config';
+
+const DEFAULT_THEMA_VALUE = 'Mijn Amsterdam algemeen';
+
+export default function useThema() {
+  return getCurrentThema();
+}
+
+export function getCurrentThema(documentTitle?: string) {
+  let title = '';
+  if (documentTitle === undefined) {
+    title = typeof document !== undefined ? document.title : '';
+  } else {
+    title = documentTitle;
+  }
+
+  return (
+    Object.values(ChapterTitles).find((t) => {
+      return title.includes(t);
+    }) ?? DEFAULT_THEMA_VALUE
+  );
+}
```

Comment:
Deze functionaliteit is zo te zien direct gerelateerd aan de analytics functionaliteit. Zou je dit niet beter in die file kunnen zetten en indien geen thema expliciet wordt meegegeven deze functionaliteit aanroepen?

bv:

```
maThema(themaNaam ?? getDerivedThemaNaamFromDocumentTitle()),
```

Zo kun je er bij non-thema-pagina downloads ook tracken. Bv onder Actueel:

```
Melding1:
  trackDownload(Uitkeringsspecificatie, thema=Chapters.INKOMEN);
Melding2:
  trackDownload(Besluit, thema=Chapters.ZORG);
```

Reply:
Ook in de `usePageChange` hook kun je deze functionaliteit weghalen en in de `trackPageViewWithCustomDimension` coden.